### DISCUSSION TITLE
doubt in pushing project to github account.

### DISCUSSION THREAD

**Discussion by Kishwaritarannum** [akash-coded/mern#16] [Q&A]
#Kishwari -> I am not able to push my project from eclipse to my github account.
@akash-coded

**Comment by akash-coded** [comment]
@Kishwaritarannum Can you please elaborate on the issue and what is your desired outcome?

**Comment by Kishwaritarannum** [comment]
Sir, I am not able to push my project from eclipse to my github account.
